.github/workflows/asciidoctor-ghpages.yml

```diff
@@ -5,8 +5,8 @@ name: GitHub Pages Publish
 on:
   push:
     branches: [ master, main ]
-  pull_request:
-    branches: [ master, main ]
+#  pull_request:
```

Comment:
Can take these out completely instead of commenting-out: we know how to trigger this behavior, and it's clearly wrong for this repository.